packages/twenty-front/src/modules/databases/hooks/useSyncRemoteTable.ts

```diff
@@ -0,0 +1,40 @@
+import { useCallback } from 'react';
```

Comment:
databases modules does not look clear to me. It looks to be part of integrations

Reply:
looks like a global change that we need to discuss with @thaisguigon! No really related to that PR